nixos/lib/make-single-disk-zfs-image.nix

```diff
@@ -244,7 +244,7 @@ let
             ${if formatOpt == "raw" then ''
             mv $rootDiskImage $out/${rootFilename}
           '' else ''
-            ${pkgs.qemu}/bin/qemu-img convert -f raw -O ${formatOpt} ${compress} $rootDiskImage $out/${rootFilename}
+            ${pkgs.qemu_kvm}/bin/qemu-img convert -f raw -O ${formatOpt} ${compress} $rootDiskImage $out/${rootFilename}
```

Comment:
It looks like runInLinuxVM actually uses `qemu`.  Maybe that should be fixed?

Reply:
to the best of my knowledge it uses `    ${qemu-common.qemuBinary qemu} \`
and `qemu = buildPackages.qemu_kvm` if I'm not wrong

Reply:
therefore qemu_kvm is good here too

Reply:
You're right, I missed the `qemu` binding.  That's confusing.